### PR TITLE
oop hometask done

### PR DESCRIPTION
**Pull request by taras-fid**

### CODE REVIEW

File: src/oop/app/src/Models/Movie.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace src\oop\app\src\Models;
+
+class Movie implements MovieInterface
+{
+
+    public string $title;
```
Comment (azmentor) [inline src/oop/app/src/Models/Movie.php]: You have public properties and the getters and setters to them. Maybe these properties should be protected or privat?

File: src/oop/app/src/Parsers/KinoukrDomCrawlerParserAdapter.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+namespace src\oop\app\src\Parsers;
+
+use src\oop\app\src\Models\Movie;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+class KinoukrDomCrawlerParserAdapter implements ParserInterface
+{
+
+    private string $title = '';
```
Comment (azmentor) [inline src/oop/app/src/Parsers/KinoukrDomCrawlerParserAdapter.php]: Do you really need these properties? You can use the local variables instead as you have only one method. But it's not a mistake, only a recommendation.

File: src/oop/app/src/Models/Movie.php (hunk 1 above)
Comment (taras-fid) [reply src/oop/app/src/Models/Movie.php]: I am apologize for this stupid mistake.
Thanks for pointing out, changed access properties in last commit.

File: src/oop/app/src/Parsers/KinoukrDomCrawlerParserAdapter.php (hunk 2 above)
Comment (taras-fid) [reply src/oop/app/src/Parsers/KinoukrDomCrawlerParserAdapter.php]: Yes, your right, thank You.
Remove class properties and rewrite method in last commit.

**Pull request merged**